app/src/main/java/com/cornellappdev/transit/ui/components/AddFavoritesSearchSheet.kt

```diff
@@ -0,0 +1,123 @@
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.Clear
+import androidx.compose.material.icons.outlined.Search
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.cornellappdev.transit.ui.theme.TransitBlue
+import com.cornellappdev.transit.ui.theme.sfProDisplayFamily
+import com.cornellappdev.transit.ui.theme.sfProTextFamily
+import com.cornellappdev.transit.ui.viewmodels.RouteViewModel
+
+/**
+ * Contents of AddFavorites BottomSheet
+ * @param routeViewModel The routeViewModel used in the app
+ * @param cancelOnClick The function to run when the cancel button is clicked
+ */
+@Composable
+fun AddFavoritesSearchSheet(
+    routeViewModel: RouteViewModel,
+    cancelOnClick: () -> Unit,
+) {
+
+
+    // Search bar flow
+    val searchBarValue = routeViewModel.searchQuery.collectAsState().value
+
+    Column(
+        modifier = Modifier
+            .fillMaxHeight(0.92f)
+            .fillMaxWidth()
+    ) {
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .background(color = Color.White),
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(10.dp),
+                horizontalArrangement = Arrangement.SpaceBetween,
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Box(modifier = Modifier.weight(1f)) {
+                    Text(
+                        text = "Add Favorites",
+                        fontFamily = sfProDisplayFamily,
+                        fontStyle = FontStyle.Normal,
+                        fontWeight = FontWeight(600),
+                        fontSize = 20.sp,
+                        modifier = Modifier.align(Alignment.Center)
+                    )
+                    TextButton(
+                        onClick = cancelOnClick,
+                        content = {
+                            Text(
+                                text = "Cancel",
+                                fontFamily = sfProTextFamily,
+                                fontStyle = FontStyle.Normal,
+                                textAlign = TextAlign.Center,
+                                fontSize = 14.sp,
+                                fontWeight = FontWeight(500),
+                                color = TransitBlue,
+                            )
```

Comment:
I think the cancel button is supposed to be some shade of gray? 
![image](https://github.com/cuappdev/ithaca-transit-compose/assets/47724806/82fd96f8-e175-45e6-923d-42115006e29b)

Ignore the arrow lol

![image](https://github.com/cuappdev/ithaca-transit-compose/assets/47724806/7dcede1b-b2dd-4dff-997d-3fb16767349d)